.github/workflows/test.yml

```diff
@@ -11,88 +11,72 @@ env:
 
-  vunit:
-    strategy:
-        fail-fast: false
-        max-parallel: 3
-        matrix:
-          task: [
-            vhpidirect/arrays/matrices/vunit_axis_vcs,
-          ]
+  win-stable:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - env:
+        WINDOWS_RELEASE: 0.37-mingw64-llvm
+      shell: bash
+      run: |
+        curl -fsSL -o ghdl.zip https://github.com/ghdl/ghdl/releases/download/v0.37/ghdl-${WINDOWS_RELEASE}.zip
+        7z x ghdl.zip "-o../ghdl-tmp" -y
+        mv ../ghdl-tmp/GHDL/${WINDOWS_RELEASE}/ ../ghdl
+        rm -rf ../ghdl-tmp ghdl.zip
+        export PATH=$PATH:$(pwd)/../ghdl/bin
+        python -m pip install pytest vunit_hdl --progress-bar off
+        python -m pytest -v -s -ra test.py --color=yes
+
+  lin-setup:
     runs-on: ubuntu-latest
     env:
       DOCKER_BUILDKIT: 1
       DOCKER_IMAGE: ghdl/vunit:llvm
     steps:
     - uses: actions/checkout@v2
-    - run: docker pull $DOCKER_IMAGE
-    - run: docker run --rm -tv $(pwd):/src -w /src/${{ matrix.task}} -e CI $DOCKER_IMAGE python3 run.py -v
+    - uses: umarcor/setup-ghdl@master
+      with:
+        backend: llvm
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        ghdl --version
+        sudo apt update -qq
+        sudo apt install -y imagemagick
+        python -m pip install --progress-bar off pytest vunit_hdl
+    - run: python -m pytest -v -s -ra test.py --color=yes
 
-  win:
-    strategy:
-        fail-fast: false
-        max-parallel: 3
-        matrix:
-          task: [
-            vhpidirect/quickstart/random,
-            vhpidirect/quickstart/math,
-            vhpidirect/quickstart/customc,
-            vhpidirect/quickstart/wrapping/basic,
-            vhpidirect/quickstart/wrapping/time,
-            #vhpidirect/quickstart/linking/bind,  ! needs investigation, output of list-link seems to have wrong path format
-            vhpidirect/quickstart/package,
-            vhpidirect/quickstart/sharedvar,
-            vhpidirect/shared/shlib,
-            #vhpidirect/shared/dlopen,  ! dlfcn.h is not available on win
-            #vhpidirect/shared/shghdl,  ! dlfcn.h is not available on win
-            vhpidirect/arrays/intvector,
-            vhpidirect/arrays/logicvector,
-            vhpidirect/arrays/matrices,
-            #vhpidirect/arrays/matrices/framebuffer, ! needs ImageMagick's 'convert'
-            vpi/quickstart,
-          ]
+  win-setup:
     runs-on: windows-latest
-    env:
-      WINDOWS_RELEASE: 0.37-mingw64-llvm
     steps:
     - uses: actions/checkout@v2
-    - shell: bash
+    - uses: eine/setup-msys2@v0
+      with:
+        msystem: MINGW64
+        update: true
+        install: mingw-w64-x86_64-python-pip
```

Comment:
The failure of this task should be fixed by adding make here: `install: mingw-w64-x86_64-python-pip make`.